sources/europe/de/NRW_ortho_wms.geojson

```diff
@@ -9,7 +9,7 @@
         "id": "nrw_ortho_wms",
         "description": "RGB-/Colorbild - Normale farbliche Darstellung.",
         "country_code": "DE",
-        "best": false,
+        "best": true,
```

Comment:
```suggestion
        "best": false,
```

How do you define "best"?

For half of NRW the `nrw_idop_wms` imagery (green area on the following screenshot) is more recent (~ 1 year) than `nrw_ortho_wms` (blue area).

So maybe don't assign "best" to any source here.

See also 
https://github.com/osmlab/editor-layer-index/pull/2022#issuecomment-2006064942
https://github.com/osmlab/editor-layer-index/issues/2273#issuecomment-2080384564

![image](https://github.com/osmlab/editor-layer-index/assets/20856381/6a380e0f-6f65-4907-9dc3-923e7d11bd52)

Reply:
Something that is missing cannot be "best". If multiple are present, then "best" is the most useful in terms of quality and currentness.

Reply:
> Something that is missing cannot be "best".

Others might argue:

- Something that is not the most recent cannot be "best".
- Something that has not the best resolution cannot be "best".

Until there is a clear definition/rule for "best" based on consensus, this is a never ending game.

As mentioned in https://github.com/osmlab/editor-layer-index/issues/2273#issuecomment-2080384564:

> Given the lack of a clear definition for what is "best", I currently won't assign "best" to any of the three sources.

Reply:
I don't see any need to open a policy debate over the attribute. Nor do I see any reason to decide anything for eternity. Surely one can simply ***update the data***, when imagery changes again in the future.

This PR merely compares `dop` and `vdop`, where clearly `vdop` is mostly empty. I would think `vdop` is clearly less valuable than `dop` at this moment. When it did have imagery, it was blurrier than `dop` too, so it was "better" in terms of age but not in terms of (optical) resolution.

This PR does not concern itself with `idop`, so please discuss that elsewhere. Do not use it to delay this PR.

Reply:
I don't like the tone in whole parts of this discussion.

> This PR does not concern itself with idop, so please discuss that elsewhere. 

IMHO you can't discuss this matter entirely without taking `idop` into account.

> Do not use it to delay this PR.

It's not my intention to delay this PR. I was asked by @tordans in https://github.com/osmlab/editor-layer-index/issues/2273#issuecomment-2080059482 for my opinion.

As mentioned in https://github.com/osmlab/editor-layer-index/issues/2273#issuecomment-2080430233:
"I don't have a strong opinion here, go with `dop` as best if you want."